deployment/helm/templates/controller.yaml

```diff
@@ -40,12 +40,12 @@ spec:
       nodeSelector:
         "kubernetes.io/os": linux
       {{- end }}
-      {{- if .Values.controller.onlyOnMaster }}
-      # Only run this pod on the master.
       tolerations:
```

Comment:
should we add another flag controller.allowOnMaster (default to true) that toggles the tolerations?

Reply:
Why wouldn't someone want the controller on master? I see it being another setting for customizing but could also see reason to wait until someone requests it

Reply:
maybe they want the master to be dedicated to Kubernetes?  maybe their master is more locked down somehow?

Reply:
That makes sense.